client/src/modules/CrossCheck/components/criteria/CrossCheckCriteria.tsx

```diff
@@ -0,0 +1,73 @@
+import { Typography } from 'antd';
+import { TaskType } from '../CrossCheckCriteriaForm';
+import { CrossCheckCriteriaDataDto } from 'api';
+
+const { Text, Title } = Typography;
+
+type Props = {
+  criteria: CrossCheckCriteriaDataDto[] | null;
+};
+
+export function CrossCheckCriteria({ criteria }: Props) {
+  const penaltyData = criteria?.filter(
+    criteriaItem => criteriaItem.type.toLocaleLowerCase() === TaskType.Penalty && criteriaItem.point,
+  );
+
+  return (
+    <>
+      {criteria
```

Comment:
consider doing early return if criteria not defined, it will remove a number of optional chaining here

Reply:
fixed